package.json

```diff
@@ -50,6 +50,8 @@
 	},
 	"devDependencies": {
 		"@cloudflare/workers-types": "^4.20221111.1",
+		"@types/dns2": "^2.0.3",
+		"dns2": "^2.1.0",
```

Comment:
these are in the monorepo's root, is that correct? shouldn't they be in the C3 package.json?
also shouldn't `dns2` be a proper dependency instead of a dev one?